### PR TITLE
add ctcc_id validation type

### PR DESCRIPTION
**Pull request by ljwoodley**

### CODE REVIEW

**Comment by ChemiKyle** [comment]
@pbchase are you satisfied with the comments?

**Comment by pbchase** [comment]
The comments look good, but the validation_name is too generic. `number_2hyp` could also descrbe an SSN. I think the validation_name should be something like `chet_ctcc_id`

File: ctcc_id.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+-- The CTCC ID will always be in the format ‘###-###-###’.
+
+-- Created for University of Florida's PDGENE project
+
+-- Created by:
+-- Laurence Woodley, UF CTSIT
+-- Kyle Chesney, UF CTSIT
+
+REPLACE into redcap_validation_types (validation_name, validation_label, regex_js, regex_php, data_type)
+VALUES ('number_2hyp', 'CHET CTCC ID (###-###-###)', '/^\d{3}-\d{3}-\d{3}$/', '/^\d{3}-\d{3}-\d{3}$/', 'text')
```
Comment (ChemiKyle) [inline ctcc_id.sql]:
```suggestion
VALUES ('chet_ctcc_id', 'CHET CTCC ID (###-###-###)', '/^\\d{3}-\\d{3}-\\d{3}$/', '/^\\d{3}-\\d{3}-\\d{3}$/', 'text')
```

When using phpmyadmin and cli mysql I had to escape the forward slash (`\`) characters preceding the digit `d` for the regex to work.

Comment (ljwoodley) [reply ctcc_id.sql]: Are we making the name change? The conversation is resolved but the changes weren't committed.

Comment (pbchase) [reply ctcc_id.sql]: I tried to commit.  I thought I had, but I don't have permission to.  @ljwoodley, could you commit the suggestion?

**Pull request merged**